backend/spec/controllers/spree/admin/resource_controller_spec.rb

```diff
@@ -64,12 +82,14 @@ def check_destroy_constraints
     Object.send(:remove_const, :Widget)
```

Comment:
Shouldn't we also remove the `MyEngine` constant at this point?

Reply:
Done, thanks!